addons/sourcemod/scripting/shavit-misc.sp

```diff
@@ -888,6 +907,18 @@ void UpdateClanTag(int client)
 	ReplaceString(sCustomTag, 32, "{tr}", sTrack);
 	ReplaceString(sCustomTag, 32, "{rank}", sRank);
 
+	result = Plugin_Continue;
+	Call_StartForward(gH_Forwards_OnClanTagChange);
```

Comment:
This needs to be put after `CS_SetClientClanTag` is called and changed to a post forward (void).